src/components/structures/RoomView.tsx

```diff
@@ -1903,6 +1903,7 @@ export class RoomView extends React.Component<IRoomProps, IRoomState> {
                                 loading={loading}
                                 joining={this.state.joining}
                                 oobData={this.props.oobData}
+                                roomId={this.state.roomId}
```

Comment:
Note that `this.state.roomId` may be undefined, so this is causing a typescript error. We'll need to consider how this should behave if we do not (yet) have a room id.

Likewise in the other places this is referenced, below.

Reply:
I changed `roomId` to be optional in `RoomPreviewBar.tsx` and `PreviewRoomNotLoggedIn` is raised when `roomId` is defined.